website/docs/guides/1.10.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,45 @@
+- You can now manage [Cluster Outage Simulations](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Cluster-Outage-Simulation/operation/startOutageSimulation) with [`mongodbatlas_cluster_outage_simulation`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cluster_outage_simulation) 
+- You can now enable [Extended Cluster Storage Sizes](https://www.mongodb.com/docs/atlas/customize-storage/#minimum-disk-capacity-to-ram-ratios) in [`mongodbatlas_project`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project). This setting allows M40+ clusters to be configured with storage greater than the standard maximum (up to 4TB for M40, 8TB for M50/M60, and 14TB for M80+). Clusters with extended storage sizes must be on AWS or GCP, and cannot span multiple regions. 
+- You can now leverage the [`PrivateEndpoint.srvShardOptimizedConnectionString`](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Multi-Cloud-Clusters/operation/createCluster) parameter in [`mongodbatlas_advanced_cluster`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/data-sources/advanced_cluster) and [`mongodbatlas_cluster`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/data-sources/cluster) Data Sources. This is a private endpoint-aware connection string optimized for sharded clusters that uses the `mongodb+srv://` protocol to connect to MongoDB Atlas.
+- You can now leverage the [`retainBackups`](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Multi-Cloud-Clusters/operation/createCluster) parameter in [`mongodbatlas_advanced_cluster`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_cluster) and [`mongodbatlas_cluster`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cluster) Resources. Set to true to retain backup snapshots for the deleted M10+ cluster. The default value is false.
+
+
+**Changes**
+- [Programmatic API Key (or PAK)](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Programmatic-API-Keys) Resources and Data Sources have been updated to make easier and more intuitive to use ([`mongodbatlas_api_key`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/api_key), [`mongodbatlas_project_api_key`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project_api_key) and [`mongodbatlas_project_ip_access_list_key`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/access_list_api_key)). We have created a dedicated [PAK Upgrade Guide](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/PAK-upgrade-guide-1.10.0) to help you transition smoothly from the previous version which was first released in Terraform Provider for MongoDB Atlas in v1.8.0 to the new version in v1.10.0.
```

Comment:
I would not use PAK for headers, only inline.   Many users will not know what PAK is, Programmatic API Key they will know.

Reply:
Makes sense, updated all headers + filename accordingly